ui/src/timeMachine/components/view_options/ScatterOptions.tsx

```diff
@@ -0,0 +1,184 @@
+  xDomain: [number, number]
+  yDomain: [number, number]
+  xAxisLabel: string
+  yAxisLabel: string
+  xPrefix: string
+  xSuffix: string
+  yPrefix: string
+  ySuffix: string
+  colors: Color[]
+  showNoteWhenEmpty: boolean
+}
+
+type Props = OwnProps & DispatchProps & StateProps
+
+const ScatterOptions: SFC<Props> = props => {
+  const {
+    fillColumns,
+    symbolColumns,
+    colors,
+    availableGroupColumns,
+    yAxisLabel,
+    xAxisLabel,
+    onSetFillColumns,
+    onSetSymbolColumns,
+    onSetColors,
+    onSetYAxisLabel,
+    onSetXAxisLabel,
+    yPrefix,
+    ySuffix,
+    onUpdateAxisSuffix,
+    onUpdateAxisPrefix,
+  } = props
+
+  const groupDropdownStatus = availableGroupColumns.length
+    ? ComponentStatus.Default
+    : ComponentStatus.Disabled
+
+  return (
+    <Grid.Column>
+      <h4 className="view-options--header">Customize Scatter Plot</h4>
+      <h5 className="view-options--header">Data</h5>
+
+      <Form.Element label="Symbol domain column">
+        <MultiSelectDropdown
+          selectedIDs={symbolColumns}
+          onChange={onSetSymbolColumns}
+          status={groupDropdownStatus}
+        >
+          {availableGroupColumns.map(columnName => (
+            <Dropdown.Item
+              id={columnName}
+              key={columnName}
+              value={{id: columnName}}
+            >
+              {columnName}
+            </Dropdown.Item>
+          ))}
+        </MultiSelectDropdown>
+      </Form.Element>
+      <Form.Element label="Fill domain column">
+        <MultiSelectDropdown
+          selectedIDs={fillColumns}
+          onChange={onSetFillColumns}
+          status={groupDropdownStatus}
+        >
+          {availableGroupColumns.map(columnName => (
+            <Dropdown.Item
+              id={columnName}
+              key={columnName}
+              value={{id: columnName}}
+            >
+              {columnName}
+            </Dropdown.Item>
+          ))}
+        </MultiSelectDropdown>
+      </Form.Element>
+      <h5 className="view-options--header">Options</h5>
+      <Form.Element label="Colors">
+        <ColorSchemeDropdown
+          value={colors.filter(c => c.type === 'scale')}
+          onChange={onSetColors}
+        />
+      </Form.Element>
+      <h5 className="view-options--header">Y Axis</h5>
+      <Form.Element label="Y Axis Label">
+        <Input
+          value={yAxisLabel}
+          onChange={e => onSetYAxisLabel(e.target.value)}
+        />
+      </Form.Element>
+      <AxisAffixes
+        prefix={yPrefix}
+        suffix={ySuffix}
+        axisName="y"
+        onUpdateAxisPrefix={prefix => onUpdateAxisPrefix(prefix, 'y')}
+        onUpdateAxisSuffix={suffix => onUpdateAxisSuffix(suffix, 'y')}
+      />
+      <h5 className="view-options--header">X Axis</h5>
+      <Form.Element label="X Axis Label">
+        <Input
+          value={xAxisLabel}
+          onChange={e => onSetXAxisLabel(e.target.value)}
+        />
+      </Form.Element>
```

Comment:
Should there be prefix/suffix settings for the x axis too? Also can the x axis settings come first?

Reply:
not yet, I can't get prefix suffix to work for time/date type axes.